tools/tasks/tsd.ts

```diff
@@ -1,7 +0,0 @@
-export = function tsd(gulp, plugins) {
```

Comment:
I think that this task is still usefull in some of cases, so I would not remove this.